usr/share/rear/layout/save/GNU/Linux/200_partition_layout.sh

```diff
@@ -356,7 +356,7 @@ Log "Saving disk partitions."
 (
     # Disk sizes
     # format: disk <disk> <sectors> <partition label type>
-    for disk in /sys/block/* ; do
+    for disk in ${ONLY_INCLUDE_DISKS[@]:-/sys/block/*} ; do
```

Comment:
There is a condition below: if `is_multipath_path`, the disk is skipped. What is causing this 40 minute delay then - is `multipath -c` so slow?

Reply:
Yes that's the main issue, we read block devices we don't care of.

Reply:
Why does `multipath -c` have to read block devices though? I would suppose that the information is cached somewhere (multipathd or the kernel device mapper driver).